cns/service/main.go

```diff
@@ -1005,8 +1014,8 @@ func InitializeCRDState(ctx context.Context, httpRestService cns.HTTPService, cn
 		}
 	}()
 
-	logger.Printf("Starting SyncHostNCVersion")
 	go func() {
```

Comment:
This also must be started only after reconciler is started. Can you add a Started flag in reconciler and control the order?